onnx/defs/experiments/defs.cc

```diff
@@ -299,8 +299,10 @@ OPERATOR_SCHEMA(Normalize)
         {"tensor(float16)", "tensor(float)", "tensor(double)"},
         "Constrain input and output types to float tensors.")
     .SetDoc(R"DOC(
-Given a matrix, apply L2-normalization along the last dimension.
-)DOC");
+Given a matrix, apply Lp-normalization along the provided axis.
+)DOC")
+    .Attr("axis", "(int64, default -1) the axis on which to apply normalization, -1 mean last axis.", AttrType::INT)
+    .Attr("p", "(float, default 2.0) the order of the normalization, only 2.0 is supported.", AttrType::FLOAT);
```

Comment:
If only 2.0 is supported, how about renaming the op to L2Normalization and remove this attribute? :-)

Reply:
I am planning to move it from experiment to core, and I want to make it generic so in the future we can add p=1.0.